packages/web-react/src/components/Icon/useIconStyleProps.ts

```diff
@@ -1,5 +1,7 @@
 import { cssVariablePrefix } from '@lmc-eu/spirit-design-tokens';
+import classNames from 'classnames';
 import { CSSProperties } from 'react';
+import { TextColors } from '../../constants';
 import { useClassNamePrefix } from '../../hooks';
 import { IconBoxSize, SpiritIconProps } from '../../types';
```

Comment:
```suggestion
import type { IconBoxSize, SpiritIconProps } from '../../types';
```

Out of scope, but while we're making some changes and if you have time to add it